client/src/Components/HomePage/MenuItemCategories/MenuItemCategory/CategorizedMenuItem/CategorizedMenuItem.jsx

```diff
@@ -17,8 +17,8 @@ export default function CategorizedMenuItem({ name, price }) {
   const loginStatus = useSelector(selectLoginStatus);
   const isOpenDialog = useSelector(selectIsOpenDialog);
 
-  const handleClickOpen = () => {
-    if (loginStatus !== REQUEST_STATUSES.succeeded) {
+  const handleClickAddToOrder = () => {
+    if (!sessionStorage.getItem("userId")) {
       dispatch(loginFormVisibilityChanged(true));
```

Comment:
Seems this logic with checking userId in session storage will be repeated quite often. I would create some wrapper around the sessionStorage that will expose 2 methods - setUserId and getUserId. getUserId will return userId from the session storage if it exists and dispatch opening login form in case it doesn't exist.
In this way we do not need to know everywhere what is the key for the user Id in storage and hide, reuse logic of appearing login form where it is needed